website/docs/basics/blocks-wrapper.md

```diff
@@ -5,3 +5,91 @@ sidebar_label: Wrapper
 ---
 
 [![docs-source](https://img.shields.io/badge/source-eigthshift--frontend--libs-yellow?style=for-the-badge&logo=javascript&labelColor=2a2a2a)](https://github.com/infinum/eightshift-frontend-libs/tree/develop/blocks/init/src/blocks/)
+
+The wrapper is our best and most powerful feature. In a nutshell, the wrapper is nothing more than a high order component (parent component). 
+
+When setting up block registration, we made, so every block view passes the true wrapper component before it is registered. This gives us the ability to have multiple shared attributes set in one component. The wrapper is designed to be the ultimate top-level component that controls how your block behaves in the website layout. By default, you can control a whole lot of stuff, but there is an ability to add your custom attributes and fine-tune wrapper to your project needs. Here are some of the features:
+
+> Some attributes are made responsive using the `Responsive` component, and you can control all attributes depending on the media breakpoints.
+
+### wrapperUse
+
+This attribute controls the usage of the wrapper component. It behaves the same way as described in [this chapter](blocks-component-in-block#i-dont-need-all-the-component-options-in-my-block).
+
+### wrapperUseShowControl
+
+This attribute controls if you are going to see options in the block editor for block use. It behaves the same way as described in [this chapter](blocks-component-in-block#i-dont-want-my-editor-to-be-able-to-change-components-options-in-my-block).
+
+### wrapperUseSimple
+
+We wrapped some of the options in a specific condition, and we call it `wrapperUseSimple`. In general, this attribute is set to `true` when you only want simplified options on your block. Natively it is used inside all of the inner blocks in the column block. Generali in all of the inner blocks, it is a good idea to simplify the wrapper.
```

Comment:
```suggestion
We wrapped some of the options in a specific condition, and we call it `wrapperUseSimple`. In general, this attribute is set to `true` when you only want simplified options on your block. Natively, it is used inside all of the inner blocks in the column block. Because we don't need wrappers inside wrappers inside wrappers (and so on). It's a good rule of thumb to use a simple wrapper in all of the inner blocks.
```